src/.babelrc.js

```diff
@@ -0,0 +1,22 @@
+const { NODE_ENV } = process.env;
+
+module.exports = {
+  presets: [
+    [
+      '@babel/env',
+      {
+        targets: {
+          browsers: ['ie >= 11']
+        },
+        exclude: ['transform-async-to-generator', 'transform-regenerator'],
```

Comment:
I noticed that `async/await` wasn't transformed - I guess this is whats doing that. Does this mean that consumers need to transpile this when they bundle their app?

Reply:
I've added the following https://github.com/nearform/graphql-hooks/issues/11#issuecomment-467922037 RE the decision to replace `async/await` with Promises.